modules/rosa-hcp-aws-privatelink-create-cluster.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-aws-privatelink-creating-cluster.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-aws-privatelink-create-cluster_{context}"]
+= Creating an AWS PrivateLink cluster
+
+You can create a private cluster with multiple availability zones (Multi-AZ) on {hcp-title} using the {product-title} (ROSA) CLI, `rosa`.
+
+[NOTE]
+====
+AWS PrivateLink is supported on existing VPCs only.
```

Comment:
This is a little confusing to me. Do you mean "preexisting" or is there some other nuance/word we could use. It seems obvious that AWS PrivateLink would only be supported on existing VPCs; it can't be supported on non-existing VPCS because they don't exist.